go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/go-tfe v0.14.0
 	github.com/hashicorp/go-version v1.2.1
 	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.3.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.6.0
```

Comment:
Did you mean for this sdk bump to slide in here? We might want to PR that separately!

Reply:
<img src="https://media.giphy.com/media/nLH7f5K1Tb1sY/giphy-downsized.gif">